Packages/com.unity.inputsystem/Documentation~/Testing.md

```diff
@@ -7,7 +7,7 @@ The Input System has built-in support for writing automated input tests. You can
 To set up a test assembly that uses the Input System's automation framework, follow these steps:
 
 1. In the `Packages/manifest.json` file of your project, `com.unity.inputsystem` must be listed in `testables`. This is necessary for test code that comes with the package to be included with test builds of your project.<br><br>You can, for example, add this after the `dependencies` property like so:
-    ```JSON
```

Comment:
Why did you remove the json tag?

Reply:
JSON doesn't seem to be a recognized language so the code block ends up garbled (see https://docs.unity3d.com/Packages/com.unity.inputsystem@1.0/manual/Testing.html).

Reply:
Try using lowercase json. Its the first example https://www.markdownguide.org/extended-syntax ;)

Reply:
Although the old code works for me in my md viewer so maybe our generator is bugged.